frontend/public/components/image-stream-tag.tsx

```diff
@@ -141,6 +148,39 @@ export const ImageStreamTagsDetails: React.SFC<ImageStreamTagsDetailsProps> = ({
           </div>
         )}
       </div>
+      <div className="co-m-pane__body-group">
+        <SectionHeading text={t('public~Manifest List')} />
```

Comment:
What do you think of "Supported Platforms" as a section header instead of "Manifest List'?

Reply:
I like it. Honestly Manifest List seemed 'technical' and not descriptive.

Reply:
I will change the text, and update as I fix the tests

Reply:
I've made the change locally, I'm now working through test failures.